src/scripts/safari/SafariItem.ts

```diff
@@ -12,34 +12,36 @@ type SafariItemWeighed = {
 class SafariItemController {
     private static list: Record<GameConstants.Region, Array<SafariItemWeighed>> = {
         [GameConstants.Region.kanto]: [
-            {item: {id: 'Razz', type: ItemType.berry}, weight: 2},
-            {item: {id: 'Nanab', type: ItemType.berry}, weight: 2},
-            {item: {id: 'Pinap', type: ItemType.berry}, weight: 2},
-            {item: {id: 'Sitrus', type: ItemType.berry}, weight: 2},
-            {item: {id: 'Oran', type: ItemType.berry}, weight: 2},
+            {item: {id: 'Razz', type: ItemType.berry}, weight: 1.95},
+            {item: {id: 'Nanab', type: ItemType.berry}, weight: 1.95},
+            {item: {id: 'Pinap', type: ItemType.berry}, weight: 1.95},
+            {item: {id: 'Sitrus', type: ItemType.berry}, weight: 1.95},
+            {item: {id: 'Oran', type: ItemType.berry}, weight: 1.95},
             {item: {id: 'Leaf Stone', type: ItemType.underground}, weight: 0.25},
             {item: {id: 'Fire Stone', type: ItemType.underground}, weight: 0.25},
             {item: {id: 'Thunder Stone', type: ItemType.underground}, weight: 0.25},
             {item: {id: 'Water Stone', type: ItemType.underground}, weight: 0.25},
             {item: {id: 'MediumRestore', type: ItemType.item}, weight: 2.25},
             {item: {id: 'LargeRestore', type: ItemType.item}, weight: 1.25},
             {item: {id: 'Protein', type: ItemType.item}, weight: 0.25},
+            {item: {id: 'RareCandy', type : ItemType.item}, weight : 0.25},
```

Comment:
This weight might be a bit too low, as it's not something gamebreaking, and we kinda made it to make the safari loot a bit more fun. The same for dream orbs.
Talk with Eugene about the weights. If you don't feel like doing it, i can just do it after it's merged

Reply:
! think making Rare Candy weight 1 in both safaris and taking the weights from the berries works well. As Jaas says, they're not very powerful